### PR TITLE
Migrated to Serenity/JS ^3.0.0-rc

### PR DESCRIPTION
**Pull request by viper3400**
@jan-molak, this would solve https://github.com/serenity-js/serenity-js/issues/1161. Turned out, that this was not that much work and we would need it as reference for documentation anyway.

Could you please create a `main` branch on this repo, review the PR and if all is fine do a merge to the new `main` branch?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "serenity-js-mocha-webdriverio-template",
-  "version": "1.0.0",
+  "version": "3.0.0",
```
Comment (jan-molak) [inline package.json:3]: Good idea 👍🏻

**Comment by jan-molak** [comment]
Hey @viper3400 - I've prepared the `main` branch. Would you mind rebasing? Thanks!

**Comment by viper3400** [comment]
Hi @jan-molak,  the rebase on `main` is done.

**Comment by jan-molak** [comment]
Looks like the build is affected by https://github.com/bahmutov/npm-install/issues/103

**Comment by jan-molak** [comment]
Merged; looks like that npm install must've been something temporary. Thanks!

**Pull request closed without merging**